frontend/src/metabase/search/components/SearchResult/components/Context.tsx

```diff
@@ -0,0 +1,31 @@
+// I think it's very likely that this is a dead codepath: RL 2023-06-21
```

Comment:
We might be able to remove this component, but I don't want to touch it in this PR. I'll make changes in the facelift PR if it's deemed to be dead